app/main.ipynb

```diff
@@ -2,17 +2,14 @@
  "cells": [
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 33,
    "outputs": [],
    "source": [
-    "import numpy as np\n",
-    "import pandas as pd"
+    "import pandas as pd\n",
+    "import pd as pd"
```

Comment:
Why did you delete `numpy ` import? What do mean `import pd as pd`? What does mean `import pd as pd`?